packages/babel-preset-env/data/shipped-proposals.js

```diff
@@ -1,14 +1,20 @@
+/* eslint sort-keys: "error" */
 // These mappings represent the syntax proposals that have been
 // shipped by browsers, and are enabled by the `shippedProposals` option.
 
 const proposalPlugins = {};
 
-const pluginSyntaxMap = new Map([
-  ["proposal-async-generator-functions", "syntax-async-generators"],
-  ["proposal-object-rest-spread", "syntax-object-rest-spread"],
-  ["proposal-optional-catch-binding", "syntax-optional-catch-binding"],
-  ["proposal-unicode-property-regex", null],
-  ["proposal-json-strings", "syntax-json-strings"],
-]);
+// use intermediary object to enforce alphabetical key order
+const pluginSyntaxObject = {
+  "proposal-async-generator-functions": "syntax-async-generators",
+  "proposal-json-strings": "syntax-json-strings",
+  "proposal-nullish-coalescing-operator": "syntax-nullish-coalescing-operator",
+  "proposal-object-rest-spread": "syntax-object-rest-spread",
+  "proposal-optional-catch-binding": "syntax-optional-catch-binding",
+  "proposal-optional-chaining": "syntax-optional-chaining",
+  "proposal-unicode-property-regex": null,
+};
 
-module.exports = { proposalPlugins, pluginSyntaxMap };
+const pluginSyntaxMap = new Map(Object.entries(pluginSyntaxObject));
```

Comment:
Unfortunately `Object.entries` is not supported in Node 6 (behind a flag), could you add a todo item and restore it to `Map` constructor? We can apply this change on Babel 8.

Reply:
Or maybe someone could write a feature request in the eslint repo to make `sort-keys` work with the `Map` and `Set` constructors :relaxed:

Reply:
Sorry for that confusion! I wasn't aware of the `Object.entries` limitation.

Reply:
@msholty-fd I didn't realize it before I read the CI error. The discussion above is still constructive that we have new eslint rules idea.